dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/contig/ContigMappingTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.dbsnpimporter.contig;
+
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+
+public class ContigMappingTest {
+
+    private static final String GENBANK_1 = "genbank_example_1";
+
+    private static final String GENBANK_2 = "genbank_example_2";
+
+    private static final String GENBANK_3 = "genbank_example_3";
+
+    private static final String REFSEQ_1 = "refseq_example_1";
+
+    private static final String REFSEQ_2 = "refseq_example_2";
+
+    private static final String REFSEQ_3 = "refseq_example_3";
+
+    private static final String GENBANK_CONTIG = "GL456213.1";
+
+    private static final String REFSEQ_CONTIG = "NT_166283.1";
+
+    private static final String REFSEQ_CONTIG_WITHOUT_SYNONYM = "NT_without_synonym";
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
This is not strictly needed. Is it a good practice to add this to the tests to be sure no exceptions are thrown?

Reply:
a previous version did use it. removing...